src/figuresClasses.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.Rectangle = exports.Circle = exports.Triangle = void 0;
+exports.getInfo = getInfo;
+var Triangle = /** @class */ (function () {
+    function Triangle(color, a, b, c) {
+        this.color = color;
+        this.a = a;
+        this.b = b;
+        this.c = c;
```

Comment:
[CODE KNOWLEDGE] - When you declare a constructor parameter with an access modifier TypeScript implicitly creates a property on the class with the same name as the parameter and assigns the parameter value to that property. You don't need to do it manually. Since this is a JavaScript file, this comment is not applicable here, but keep it in mind for TypeScript files.